src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -274,15 +282,26 @@ public DataAccessRequest createProgressReport(User user, DataAccessRequest progr
   }
 
   public void validateProgressReport(User user, DataAccessRequest progressReport, DataAccessRequest parentDar) {
-    validateDar(user, progressReport);
-    if (parentDar.getDraft()) {
+    validateCommonDarAndProgressReportElements(user, progressReport);
+    List<String> errorSummary = new ArrayList<>();
+    getErrorSummary(progressReport.getData().getInternalCollaborators().stream().map(Collaborator::getEmail).toList(), user.getInstitution(),
+        INTERNAL_COLLABORATOR + " " + MEMBER + ": ", INTERNAL_COLLABORATOR + " " + MEMBERS, errorSummary);
+    getErrorSummary(progressReport.getData().getLabCollaborators().stream().map(Collaborator::getEmail).toList(), user.getInstitution(),
+        LAB_STAFF + " " + MEMBER + ": ", LAB_STAFF + " " + MEMBERS, errorSummary);
+
+    if (!errorSummary.isEmpty()) {
+      throw new BadRequestException( ALL_LISTED_PERSONNEL_MUST_SHARE_THE_SAME_INSTITUTION
+          + String.join(", ", errorSummary));
+    }
+
+    if (Boolean.TRUE.equals(parentDar.getDraft())) {
       throw new BadRequestException(
           "Cannot create a progress report for a draft Data Access Request");
     }
     if (progressReport.getDatasetIds() == null || progressReport.getDatasetIds().isEmpty() ) {
       throw new BadRequestException("At least one dataset is required");
     }
-    if (!parentDar.getDatasetIds().containsAll(progressReport.getDatasetIds())) {
+    if (!new HashSet<>(parentDar.getDatasetIds()).containsAll(progressReport.getDatasetIds())) {
```

Comment:
You can use `Set.copyOf()` here instead of `new HashSet<>()`
```suggestion
    if (!Set.copyOf(parentDar.getDatasetIds()).containsAll(progressReport.getDatasetIds())) {
```